aptos-move/aptos-aggregator/src/resolver.rs

```diff
@@ -73,10 +73,6 @@ pub mod test_utils {
             Ok(self.0.get(state_key).cloned())
         }
 
-        fn is_genesis(&self) -> bool {
```

Comment:
I think you need to rebase? `is_genesis` has been removed.

Reply:
Done.